pkg/tsdb/cloudmonitoring/cloudmonitoring.go

```diff
@@ -34,11 +34,26 @@ var (
 )
 
 var (
-	matchAllCap       = regexp.MustCompile("(.)([A-Z][a-z]*)")
-	legendKeyFormat   = regexp.MustCompile(`\{\{\s*(.+?)\s*\}\}`)
-	metricNameFormat  = regexp.MustCompile(`([\w\d_]+)\.(googleapis\.com|io)/(.+)`)
-	wildcardRegexRe   = regexp.MustCompile(`[-\/^$+?.()|[\]{}]`)
-	alignmentPeriodRe = regexp.MustCompile("[0-9]+")
+	matchAllCap                 = regexp.MustCompile("(.)([A-Z][a-z]*)")
+	legendKeyFormat             = regexp.MustCompile(`\{\{\s*(.+?)\s*\}\}`)
+	metricNameFormat            = regexp.MustCompile(`([\w\d_]+)\.(googleapis\.com|io)/(.+)`)
+	wildcardRegexRe             = regexp.MustCompile(`[-\/^$+?.()|[\]{}]`)
+	alignmentPeriodRe           = regexp.MustCompile("[0-9]+")
+	cloudMonitoringUnitMappings = map[string]string{
```

Comment:
after changing the mapping, I am not able to get the same result as frontend mapping. The difference is visible on the UI.
before change i got Mb/s
![before_change](https://user-images.githubusercontent.com/74549700/104001492-07a80080-51a0-11eb-99be-942e6d24f4f8.PNG)

After change I got MiB
![after_change](https://user-images.githubusercontent.com/74549700/104001501-0971c400-51a0-11eb-91ea-a19db9fe5f9f.PNG)